lnwallet/interface.go

```diff
@@ -660,6 +660,13 @@ func InternalKeyForAddr(wallet WalletController, netParams *chaincfg.Params,
 		return none, nil
 	}
 
+	// Imported addresses do not provide private keys, so they do not
+	// implement waddrmgr.ManagedPubKeyAddress. See RPC ImportTapscript.
+	// Fix https://github.com/lightninglabs/loop/issues/923
```

Comment:
nit: I think it's okay to have the link to the issue that's fixed in the commit message (for additional context if we ever need it), but perhaps not in the code/comment itself?

Reply:
I removed the link to the issue here and in itest code as well.